.gitignore

```diff
@@ -24,6 +24,8 @@ integration-tests/logs
 integration-tests/smoke/logs
 integration-tests/soak/logs
 tmp-manifest-*
-artifacts
+/artifacts
 .local-mock-server
+codegen
+devAccounts.json
```

Comment:
json file that contains test accounts for local wasmd -- is created everytime local wasmd is setup in the script (see other code below)